Detectors/Upgrades/ALICE3/TRK/simulation/src/Detector.cxx

```diff
@@ -130,38 +106,29 @@ void Detector::configITS(Detector* its)
     its->defineWrapperVolume(iw, wrpRMin[iw], wrpRMax[iw], wrpZSpan[iw]);
   }
 
-  // Build OB only (4 layers)
-  // for (int idLr = mNumberOfInnerLayers; idLr < mTotalNumberOfLayers; idLr++) {
-  //   int im = idLr - mNumberOfInnerLayers + kNLrInner;
-  //   rLr = tdr5dat[im][kRmd];
-  //   phi0 = tdr5dat[im][kPhi0];
-
-  //   nStaveLr = TMath::Nint(tdr5dat[im][kNStave]);
-  //   nModPerStaveLr = TMath::Nint(tdr5dat[im][kNModPerStave]);
-  //   int nChipsPerStaveLr = nModPerStaveLr;
-  //   its->defineLayer(idLr, phi0, rLr, nStaveLr, nModPerStaveLr, ChipThicknessOB, Segmentation::SensorLayerThickness,
-  //                    kSensTypeID, kBuildLevel);
-  // }
-
-  // From Mario Sitta's hack
-  std::vector<std::array<double, 2>> tdr5data;
-  tdr5data.emplace_back(std::array<double, 2>{1.8f, getDetLengthFromEta(1.44f, 1.8f)});
-  tdr5data.emplace_back(std::array<double, 2>{2.8f, getDetLengthFromEta(1.44f, 2.8f)});
-  tdr5data.emplace_back(std::array<double, 2>{3.8f, getDetLengthFromEta(1.44f, 3.8f)});
-  tdr5data.emplace_back(std::array<double, 2>{8.0f, getDetLengthFromEta(1.44f, 8.0f)});
-  tdr5data.emplace_back(std::array<double, 2>{20.0f, getDetLengthFromEta(1.44f, 20.0f)});
-  tdr5data.emplace_back(std::array<double, 2>{25.0f, getDetLengthFromEta(1.44f, 25.0f)});
-  tdr5data.emplace_back(std::array<double, 2>{40.0f, getDetLengthFromEta(1.44f, 40.0f)});
-  tdr5data.emplace_back(std::array<double, 2>{55.f, getDetLengthFromEta(1.44f, 55.f)});
-  tdr5data.emplace_back(std::array<double, 2>{80.0f, getDetLengthFromEta(1.44f, 80.0f)});
-  tdr5data.emplace_back(std::array<double, 2>{100.f, getDetLengthFromEta(1.44f, 100.f)});
-
-  std::array<float, 10> sensorThicknesses = {50.e-4, 50.e-4, 50.e-4, 50.e-3, 50.e-3, 50.e-3, 50.e-3, 50.e-3, 50.e-3, 50.e-3};
+  // Layout updated at December 18th 2020 [cm]
+
+  std::vector<std::array<double, 2>> layers;
+  const int nLayers{12};
+  layers.emplace_back(std::array<double, 2>{0.5f, 15.f});
+  layers.emplace_back(std::array<double, 2>{1.2f, 15.f});
+  layers.emplace_back(std::array<double, 2>{2.5f, 15.f});
+  layers.emplace_back(std::array<double, 2>{3.75f, 62.f});
+  layers.emplace_back(std::array<double, 2>{7.f, 62.f});
+  layers.emplace_back(std::array<double, 2>{12.f, 62.f});
+  layers.emplace_back(std::array<double, 2>{20.f, 62.f});
+  layers.emplace_back(std::array<double, 2>{30.f, 62.f});
+  layers.emplace_back(std::array<double, 2>{45.f, 132.f});
+  layers.emplace_back(std::array<double, 2>{60.f, 132.f});
+  layers.emplace_back(std::array<double, 2>{80.f, 132.f});
+  layers.emplace_back(std::array<double, 2>{100.f, 132.f});
```

Comment:
Can we make this information retrievable, e.g. by adding getters to `mLayerRadii`, `mLayerZLen`, ... ? Or do you see any problem with doing that?

Reply:
True, I alsways manage to mess with this.

Reply:
Well, this class will be eventually cleaned up a lot.
Getters are no-issue, point is that as this is configured at runtime, either you access them at simulation time (where you have already the information) or does not make much sense.
When you want to know that at runtime, perhaps, is more consistent to get the quotas from some product of the simulation process, such as the geometry file which is produced by simulation.

Reply:
Not for this PR, but is there an example how this information can be retrieved at analysis time, or more specifically in the tracking macro?